src/Symfony/Bridge/Doctrine/Tests/ManagerRegistryTest.php

```diff
@@ -44,7 +44,8 @@ public function testResetService()
         $registry->resetManager();
 
         $this->assertSame($foo, $container->get('foo'));
-        $this->assertObjectNotHasAttribute('bar', $foo);
+        $this->assertInstanceOf(\stdClass::class, $foo);
+        $this->assertTrue(!isset($foo->bar));
```

Comment:
Not sure if that makes a difference here, but the more precise check would be `property_exists()`. `$foo->bar = null;` would have been caught by the old assertion, but it passes the `!isset()` check.

Reply:
indeed, makes sense